web-impl/impl/src/java/org/sakaiproject/web/impl/WebServiceImpl.java

```diff
@@ -422,10 +422,12 @@ public void transferCopyEntities(String fromContext, String toContext, List ids)
 							    for (ToolConfiguration config: toolConfs) {
 								if (config.getToolId().equals(TOOL_ID)) {
 								    SitePage p = config.getContainingPage();
+									// Checking if the 'source' property of fromSite and toSite are same by replacing respective siteIds with a constant.
+									String fromContentUrl = contentUrl.replace(fromContext, WEB_CONTENT_URL_PROP);
+									String toContentUrl = config.getPlacementConfig().getProperty(WEB_CONTENT_URL_PROP).replace(toContext,WEB_CONTENT_URL_PROP);
```

Comment:
isn't this method called only for webcontent link type of page  which always will have a url?